### DISCUSSION TITLE
in torchvision.datasets.FashionMNIST, how to check the class attribute

### DISCUSSION THREAD

**Discussion by bubble114** [mrdbourke/pytorch-deep-learning#702] [Q&A]
in the course 03. PyTorch Computer Vision, we get train_data = datasets.FashionMNIST, then len(train_data.data), len(train_data.targets), len(test_data.data), len(test_data.targets) and class_names = train_data.classes.
but how can I know these properties?
where can I find? Although I can see when I type "dot" after train_data.
Thanks

**Comment by shisirkha** [comment]
I think you need to learn more about python and its syntax.
then you will be clear why this happen.
